app/frontend/src/features/map/ClusteredUsers.ts

```diff
@@ -0,0 +1,70 @@
+import { Map as MapboxMap } from "mapbox-gl";
+import { URL } from "../../service/client";
+
```

Comment:
Minor nitpick: since this file doesn't have any React components, you should name the file in camelCase instead.